ext/mbstring/libmbfl/filters/emoji2uni.h

```diff
@@ -2,1225 +2,1234 @@ static const int mb_tbl_code2uni_docomo1_min = 0x28c2;
-	0xf17f,	0xf18e,	0xf192,	0xf194,
-	0xf195,	0xf197,	0xf199,	0xf19a,
-	0xf1e8,	0xf1e9,	0xf1ea,	0xf1eb,
-	0xf1ec,	0xf1ee,	0xf1ef,	0xf1f0,
-	0xf1f7,	0xf1fa,	0xf201,	0xf202,
-	0xf21a,	0xf22f,	0xf233,	0xf235,
-	0xf236,	0xf237,	0xf238,	0xf239,
-	0xf23a,	0xf250,	0xf300,	0xf302,
-	0xf303,	0xf304,	0xf305,	0xf306,
-	0xf307,	0xf308,	0xf30a,	0xf319,
-	0xf31f,	0xf334,	0xf335,	0xf337,
-	0xf338,	0xf339,	0xf33a,	0xf33b,
-	0xf33e,	0xf340,	0xf341,	0xf342,
-	0xf343,	0xf345,	0xf346,	0xf349,
-	0xf34a,	0xf34e,	0xf353,	0xf354,
-	0xf358,	0xf359,	0xf35a,	0xf35b,
-	0xf35c,	0xf35d,	0xf35e,	0xf35f,
-	0xf361,	0xf362,	0xf363,	0xf366,
-	0xf367,	0xf370,	0xf371,	0xf372,
-	0xf373,	0xf374,	0xf375,	0xf376,
-	0xf378,	0xf37a,	0xf37b,	0xf380,
-	0xf381,	0xf382,	0xf383,	0xf384,
-	0xf385,	0xf386,	0xf387,	0xf388,
-	0xf389,	0xf38c,	0xf38d,	0xf38e,
-	0xf38f,	0xf390,	0xf391,	0xf392,
-	0xf393,	0xf3a1,	0xf3a2,	0xf3a4,
-	0xf3a5,	0xf3a6,	0xf3a7,	0xf3a8,
-	0xf3a9,	0xf3ab,	0xf3ac,	0xf3af,
-	0xf3b0,	0xf3b1,	0xf3b5,	0xf3b6,
-	0xf3b7,	0xf3b8,	0xf3ba,	0xf3be,
-	0xf3bf,	0xf3c0,	0xf3c1,	0xf3c3,
-	0xf3c4,	0xf3c6,	0xf3c8,	0xf3ca,
-	0xf3e0,	0xf3e2,	0xf3e3,	0xf3e5,
-	0xf3e6,	0xf3e7,	0xf3e8,	0xf3e9,
-	0xf3ea,	0xf3eb,	0xf3ec,	0xf3ed,
-	0xf3ef,	0xf3f0,	0xf40d,	0xf40e,
-	0xf411,	0xf412,	0xf414,	0xf417,
-	0xf418,	0xf419,	0xf41a,	0xf41b,
-	0xf41f,	0xf420,	0xf424,	0xf426,
-	0xf427,	0xf428,	0xf42b,	0xf42c,
-	0xf42d,	0xf42e,	0xf42f,	0xf430,
-	0xf431,	0xf433,	0xf434,	0xf435,
-	0xf436,	0xf437,	0xf438,	0xf439,
-	0xf43a,	0xf43b,	0xf440,	0xf442,
-	0xf443,	0xf444,	0xf446,	0xf447,
-	0xf448,	0xf449,	0xf44a,	0xf44b,
-	0xf44c,	0xf44d,	0xf44e,	0xf44f,
-	0xf450,	0xf451,	0xf452,	0xf454,
-	0xf455,	0xf457,	0xf458,	0xf459,
-	0xf45c,	0xf45f,	0xf460,	0xf461,
-	0xf462,	0xf463,	0xf466,	0xf467,
-	0xf468,	0xf469,	0xf46b,	0xf46e,
-	0xf46f,	0xf471,	0xf472,	0xf473,
-	0xf474,	0xf475,	0xf476,	0xf477,
-	0xf478,	0xf47b,	0xf47c,	0xf47d,
-	0xf47e,	0xf47f,	0xf480,	0xf481,
-	0xf482,	0xf483,	0xf484,	0xf485,
-	0xf486,	0xf487,	0xf488,	0xf489,
-	0xf48a,	0xf48b,	0xf48d,	0xf48e,
-	0xf48f,	0xf490,	0xf491,	0xf492,
-	0xf493,	0xf494,	0xf497,	0xf498,
-	0xf499,	0xf49a,	0xf49b,	0xf49c,
-	0xf49d,	0xf49f,	0xf4a1,	0xf4a2,
-	0xf4a3,	0xf4a4,	0xf4a6,	0xf4a8,
-	0xf4a9,	0xf4aa,	0xf4b0,	0xf4b1,
-	0xf4b9,	0xf4ba,	0xf4bb,	0xf4bc,
-	0xf4bd,	0xf4bf,	0xf4c0,	0xf4d6,
-	0xf4dd,	0xf4e0,	0xf4e1,	0xf4e2,
-	0xf4e3,	0xf4e9,	0xf4eb,	0xf4ee,
-	0xf4f1,	0xf4f2,	0xf4f3,	0xf4f4,
-	0xf4f6,	0xf4f7,	0xf4fa,	0xf4fb,
-	0xf4fc,	0xf50a,	0xf50d,	0xf511,
-	0xf512,	0xf513,	0xf514,	0xf51d,
-	0xf51e,	0xf525,	0xf528,	0xf52b,
-	0xf52f,	0xf530,	0xf531,	0xf532,
-	0xf533,	0xf534,	0xf550,	0xf551,
-	0xf552,	0xf553,	0xf554,	0xf555,
-	0xf556,	0xf557,	0xf558,	0xf559,
-	0xf55a,	0xf55b,	0xf5fb,	0xf5fc,
-	0xf5fd,	0xf601,	0xf602,	0xf603,
-	0xf604,	0xf609,	0xf60a,	0xf60c,
-	0xf60d,	0xf60f,	0xf612,	0xf613,
-	0xf614,	0xf616,	0xf618,	0xf61a,
-	0xf61c,	0xf61d,	0xf61e,	0xf620,
-	0xf621,	0xf622,	0xf623,	0xf625,
-	0xf628,	0xf62a,	0xf62d,	0xf630,
-	0xf631,	0xf632,	0xf633,	0xf637,
-	0xf645,	0xf646,	0xf647,	0xf64c,
-	0x
```

Comment:
same as before: missing final single EOL character.

Reply:
dry shot. will try again tommorow.

Reply:
Worth installing editorconfig plugin for your editor or IDE and it's done automatically as it should be done...

Reply:
The `.editorconfig` file from the php-src project root directory takes care of that from now on. Also don't forget to probably reload the editor/IDE.